app/controllers/admin/assets_controller.rb

```diff
@@ -54,6 +64,11 @@ def fixity_report
 
   def display_attach_form
     @parent = Work.find_by_friendlier_id!(params[:parent_id])
+    if @parent.genre && @parent.genre.include?('Oral histories') && @parent.published?
```

Comment:
Should this re-use the logic from `published_oral_history_asset?`like the others, or is this different?

Reply:
oh nevermind, I see that this one is based on @parent, which is different. It could be DRY'd but I'm not gonna worry about it.